server/master/src/main/java/org/apache/accumulo/master/tableOps/BulkImport.java

```diff
@@ -266,6 +277,26 @@ public Exception call() throws Exception {
         throw ex.get();
       }
     }
+
```

Comment:
Suggest using a serialization framework here to serialize an object to the file. Is GSON or JAXB on the classpath in 1.10?

Reply:
I think they both are.